bon-macros/src/builder/builder_gen/input_func.rs

```diff
@@ -503,3 +504,17 @@ impl Visit<'_> for FindSelfReference {
         }
     }
 }
+
+fn get_must_use_attribute(attrs: &[syn::Attribute]) -> darling::Result<Option<syn::Attribute>> {
```

Comment:
The `Result` type of the crate is already in scope via `crate::util::prelude::*` import, no need to qualify it.
```suggestion
fn get_must_use_attribute(attrs: &[syn::Attribute]) -> Result<Option<syn::Attribute>> {
```

Reply:
Ugh, Result ≠ Result… I dislike prelude, wildcard imports and shadowing the std/core prelude as it quickly gets confusing. :D